server/services/stats/side/test.go

```diff
@@ -65,6 +72,30 @@ func (t *testNrClient) GetServersStats() ([]newrelic.Server, error) {
 	return servers, nil
 }
 
+type testTcClient struct {
+	CountGetTask int8
+}
+
+func NewTestTcClient() TcClient {
+	return &testTcClient{
+		CountGetTask: 5,
+	}
+}
+
+func (t *testTcClient) GetBuildStats() ([]*teamcity.Build, error) {
+	tasks := make([]*teamcity.Build, t.CountGetTask, t.CountGetTask)
```

Comment:
I use make to initialize the slice of the required length and capacity. If using the append, then, several arrays are initialized, doubled the depth. It extra garbage in the memory. We can avoid this by knowing the needed size